lib/nano.js

```diff
@@ -553,6 +553,9 @@ module.exports = exports = nano = function dbScope(cfg) {
       }
       qs = qs || {};
 
+      // prevent mutation of the client qs object by using a clone
+      qs = JSON.parse(JSON.stringify(qs))
```

Comment:
Could we rather use `Object.assign` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
we could. Change on the way.